image-layout.md

```diff
@@ -44,11 +44,16 @@ afff3924849e458c5ef237db5f89539274d5e609db5db935ed3959c90f1f2d51 ./blobs/sha256/
 ```
 
 Object names in the `refs` subdirectories MUST NOT include characters outside of the set of "A" to "Z", "a" to "z", "0" to "9", the hyphen `-`, the dot `.`, and the underscore `_`.
+
 Object names in the `blobs` subdirectories are composed of a directory for each hash algorithm, the children of which will contain the actual content.
+
 A blob, referenced with digest `<alg>:<hex>` (per [descriptor](descriptor.md#digests-and-verification)), MUST have its content stored in a file under `blobs/<alg>/<hex>`.
+
 The character set of the entry name for `<hex>` and `<alg>` MUST match the respective grammar elements described in [descriptor](descriptor.md#digests-and-verification).
 For example `sha256:5b` will map to the layout `blobs/sha256/5b`.
```

Comment:
I think you *should* be separating one big paragraph [into four][1].  With 833917c3 you're separating it into six.  The [`blob`-tree paragraph][1] I'd prefer would contain several sentences: two for the `blobs/<algo>/<hash>` structure, one for the algo and hash charsets, and one example.  I think those are all closely-related enough to belong in the same paragraph.

[1]: https://github.com/opencontainers/image-spec/pull/373#discussion_r82899107